src/object/srv_obj.c

```diff
@@ -1452,15 +1452,15 @@ obj_ioc_init(uuid_t pool_uuid, uuid_t coh_uuid, uuid_t cont_uuid, int opc,
 	D_DEBUG(DB_TRACE, DF_UUID"/%p is rebuild cont hdl\n",
 		DP_UUID(coh_uuid), coh);
 
-	/* load csummer on demand for rebuild - will be destroyed in
-	 * obj_ioc_fini if rebuild container
-	 */
-	rc = ds_cont_csummer_init(&coh->sch_csummer, pool_uuid, cont_uuid);
+	/* load VOS container on demand for rebuild */
+	rc = ds_cont_child_lookup(pool_uuid, cont_uuid, &coc);
 	if (rc)
 		D_GOTO(failed, rc);
 
-	/* load VOS container on demand for rebuild */
-	rc = ds_cont_child_lookup(pool_uuid, cont_uuid, &coc);
+	/* load csummer on demand for rebuild - will be destroyed in
+	 * obj_ioc_fini if rebuild container
```

Comment:
It's not necessary to destroy it on obj_ioc_fini(), otherwise, you have to initialize it on every io request?

Reply:
removed